butterknife/src/main/java/butterknife/ButterKnife.java

```diff
@@ -83,7 +85,7 @@ private ButterKnife() {
     throw new AssertionError("No instances.");
   }
 
-  /** An unbinder contract that can be bind with {@link butterknife.Unbinder}. */
+  /** An unbinder contract that will unbind views when called. */
   @SuppressWarnings("unused") // Used by generated code.
   public interface ViewUnbinder<T> {
```

Comment:
Ah you're right. I was stupidly trying to remove all parametrization rather than just the initial `implements` form. The inner unbinders still need to be parametrized, but the original implements does not. Fixed in c4f4aaa